src/components/header.js

```diff
@@ -0,0 +1,34 @@
+import OpenModal from "./about-modal";
+
+export default function Header() {
+  let feedStart = new Date().toLocaleTimeString();
+  return (
+    <nav className="bg-gray-800">
+      <div className="mx-auto max-w-7xl">
+        <div className="h-16 md:flex md:items-center md:justify-between md:space-x-5">
+          <div className="flex items-center space-x-5">
+            <div className="flex-shrink-0">
+              <div className="relative">
+                <img src="/logo192.png" alt="Eddie Hub Logo" height={38} width={38} />
+                <span className="absolute inset-0 rounded-full shadow-inner" aria-hidden="true" />
+              </div>
+            </div>
+            {/*
+            Use vertical padding to simulate center alignment when both lines of text are one line,
+            but preserve the same layout if the text wraps without making the image jump around.
+          */}
```

Comment:
Should we leave the comment in?

Reply:
no, that was tailwind boilerplate.  I'll remove it. Good eye!